lib/class-wp-rest-oauth1-ui.php

```diff
@@ -166,7 +166,7 @@ public function handle_callback_redirect( $verifier ) {
 		$args = urlencode_deep( $args );
 		$callback = add_query_arg( $args, $callback );
 
-		wp_redirect( $callback );
+		wp_safe_redirect( $callback );
```

Comment:
This one can't use safe redirection, as it's not on the site itself. All the rest look good. :+1: